docs/content/languages/python.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Using Python with Fission"
+weight: 10
+---
+
+Fission supports functions written in Python.  Both Python 2.x and
+Python 3.x are supported.  In this usage guide we'll cover how to use
+this environment, write functions, and work with dependencies.
+
+## Before you start
+
+We'll assume you have Fission and Kubernetes setup.  If not, head over
+to the [install guide]().  Verify your Fission setup with:
+
+```
+fission --version
+```
+
+## Add the Python environment to your cluster
+
+Fission language support is enabled by creating an _Environment_.  An
+environment is the language-specific part of Fission.  It has a
+container image in which your function will run.
+
+```
+fission environment create --name python --image fission/python-env --builder fission/python-builder
+```
+
+## Write a simple function in Python
+
+TODO hello world
+
+## HTTP requests and HTTP responses
+
+### Accessing HTTP Requests
+
+#### Headers
+
+TODO example of getting headers
+
+#### Query string
+
+TODO example of accessing query string
+
+#### Body 
+
+TODO example of accessing body 
+-- as plain text 
+-- as json
+
+### Controlling HTTP Responses 
+
+#### Setting Response Headers
+
+TODO
+
+#### Setting Status Codes 
+
+TODO
+
+## Working with dependencies
```

Comment:
Maybe file this section under a broader term `advanced functionality`, `advanced features`, because not every environment has the same features (such as a builder), and there are probably more topics we want to cover (adding environment variables, adding storage, working with message queues, etc.).

Reply:
Unrelated: what about a matrix somewhere in the guide to give users a single reference to all the info the user needs. Something like:

k                          | v
---------------- | -------------------
Runtime image   | fission/python-env                      
Builder image     | fission/python-builder
Env version         | 3
Supports multifile | yes

This table would have the same keys in every environment.

Reply:
The table is a good idea for a consistent at-a-glance look.  I like it but I think the keys column needs some work.  I'd like to get to this in a future release.  For now I'd like us to get _some_ docs out the door.